content/post/0-to-60-analytics-stack.md

```diff
@@ -0,0 +1,79 @@
+
+For finance, again, different providers have different billing structures and you may not realize how much of a sticking point this can be. Additionally, your company may want to stick with the default analytics DB of the cloud provider you are already using. However, unlike data moving, there are substantial feature a differences between paid options. You should be prepared to advocate for your preferred solution by articulating the value add in terms of faster project throughput and/or less human intervention costs.
+
+
+## Business Intelligence & Visualization:
+
+- Some solutions you might consider: AWS Quicksight, Periscope, Mode, Looker, Holistic, Metabase (Free), Superset (Free)
+
+Even though the BI purchase is expensive, it's often the least contentious  since it is the only technology of the three that business people actually see! Build a value proposition by combining expected maintenance costs (in salary/opportunity cost) with the value of avoiding a bad decision due to bad data ($10K?). By my simple back of the envelope calculations, Looker ended up being cheaper than (free) Superset, because it largely runs itself and enables self-serve.
+
+This is one of the purchase decisions that may actually merit a deck or formal document outlining the pros and cons of each option. However, I regret over-democratizing the process by involving multiple business users to help make the decision by committee. At the end of the day, making simple dashboards is similar in each system, and it is you, not business users, who will bear the burden of maintaining permissions, query accuracy, and slack integrations. Reach out to the greater data community and hold yourself accountable to make the right decision.
+
+I made one specific mistake here that I want to share: Looker was much more expensive than the other options, and, to explain the cost, I leaned on the value proposition of teams not needing to hire a dedicated analyst to write SQL queries. This was a mistake for two reasons:
+
+1.  Business users still need a baseline skill level to use Looker. A surprisingly effective rule-of-thumb is that if a person is comfortable with pivot tables in excel, they can do Looker. Do a quick calculation of how many people in your org need data, and of those, how many of them you would trust to make a pivot table.
+
+2. There are benefits to hiring an analyst beyond simply doing SQL queries. Looker saved us from hiring an analyst, but that analyst could also have added value by connecting the dots of cross-functional business requests into product priorities. Unless you are severely budget constrained, it doesn’t hurt to have another bright technical person around, and the earlier you hire them, the better.
```

Comment:
In my experience tools like Looker don't reduce the need for analysts, they just change the type of requests you get. When we rolled out Looker we actually started to get more requests. Now the requests had evolved from refreshing recurring reports to customizing Looker dashboards and digging into the why.